src/main/java/org/gridsuite/gateway/endpoints/CaseServer.java

```diff
@@ -0,0 +1,35 @@
+/*
+  Copyright (c) 2021, RTE (http://www.rte-france.com)
```

Comment:
all copyright from new files should be 2022

Reply:
Created in 2021 but pushed to 2022